src/Prima/Pyxis/Form/Validation.elm

```diff
@@ -23,21 +49,51 @@ import Regex
 {-| Represents a validation entry.
 -}
 type Validation model
-    = NotEmpty String
-    | Expression Regex.Regex String
-    | Custom (model -> Bool) String
+    = NotEmpty ValidationType String
+    | Expression ValidationType Regex.Regex String
+    | Custom ValidationType (model -> Bool) String
+
+
+{-| Represents a validation type.
+-}
+type ValidationType
+    = Error
+    | Warning
+
+
+{-| Returns ValidationType Error.
+-}
+severityError : ValidationType
+severityError =
+    Error
+
+
+{-| Returns ValidationType Waring.
+-}
+severityWarning : ValidationType
+severityWarning =
```

Comment:
non vedo molto il senso di queste due funzioni, che potrebbero essere sostituite direttamente con il costruttore. Occhio al typo in _Waring_ nel commento

Reply:
si, si potrebbe esporre il tipo direttamente.
Sarebbe anche più carino da usare NotEmpty severityError "Empty value is not acceptable."

Reply:
o SeverityLevel

Reply:
@matteosister quindi dici di usare direttamente il costruttore? Per me va bene, @lucabell ti sembra più espressivo? A pensarci in effetti a me convince...